package.json

```diff
@@ -6,12 +6,13 @@
     ],
     "scripts": {
         "build": "yarn tsc -b",
+        "package": "yarn tsup packages/json-react-layouts/src/index.ts --external debug --external react --target node14 --format cjs,esm --legacy-output -d packages/json-react-layouts/dist && yarn tsc -p packages/json-react-layouts --declaration --outDir packages/json-react-layouts/dist --emitDeclarationOnly && yarn tsup packages/json-react-layouts-data-loader/src/index.tsx --external debug --external react --target node14 --format cjs,esm --legacy-output -d packages/json-react-layouts-data-loader/dist && yarn tsc -p packages/json-react-layouts-data-loader --declaration --outDir packages/json-react-layouts-data-loader/dist --emitDeclarationOnly",
```

Comment:
Is it possible to use `tsup --dts` to generate typedefs at build time? As per https://tsup.egoist.sh/#what-about-type-checking

If so, are there any advantages of `tsc -d --emitDeclarationOnly` over `tsup --dts`?

Reply:
I can't get it to build using this option

Reply:
I had issues when doing our nx-generator and this is the approach I used in the end there too